Integrations/FortiGate/FortiGate.yml

```diff
@@ -437,6 +437,38 @@ script:
       description: Was address group deleted
       type: boolean
     description: Delete an address group from FortiGate firewall
+  - name: fortigate-ban-ip
+    arguments:
+    - name: ip_addresses
+      required: true
+      description: List of IP Addresses to ban, comma separated. IPv4 and IPv6 addresses
+        are allowed.
+    - name: expiry
+      description: Time until expiry in seconds. 0 for indefinite ban.
+    description: Immediately add one or more users to the banned list.
```

Comment:
IPs, not users

Reply:
It's taken directly from their documentation - IPs and users are the same according to them